docs/connectors/sources/debezium.md

```diff
@@ -0,0 +1,226 @@
+          "database.hostname": "[POSTGRES HOST NAME]",
+          "database.port": "[POSTGRES PORT]",
+          "database.user": "[DEBEZIUM USERNAME]",
+          "database.password": "[DEBEZIUM PASSWORD]",
+          "database.dbname": "[DATABASE NAME]",
+          "table.include.list": "[TABLE LIST]",
+          "topic.prefix": "[KAFKA TOPIC PREFIX]",
+          "key.converter": "io.confluent.connect.avro.AvroConverter",
+          "value.converter": "io.confluent.connect.avro.AvroConverter",
+          "key.converter.schemas.enable": "true",
+          "value.converter.schemas.enable": "true",
+          "key.converter.schema.registry.url": [SCHEMA REGISTRY URL],
+          "value.converter.schema.registry.url": [SCHEMA REGISTRY URL]
+      }
+  }'
+```
+
+Create a Debezium connector to read changes from a MySQL database into JSON-encoded Kafka topics:
+
+```
+curl -i -X \
+    POST -H "Accept:application/json" -H "Content-Type:application/json" \
+    [KAFKA CONNECT HOSTNAME:PORT]/connectors/ -d \
+    '{ "name": "my-connector",
+        "config": {
+            "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+            "tasks.max": "1",
+            "database.hostname": "[MYSQL HOST NAME]",
+            "database.port": "[MYSQL PORT]",
+            "database.user": "[DEBEZIUM USERNAME]",
+            "database.password": "[DEBEZIUM PASSWORD]",
+            "database.server.id": "[UNIQUE DATABASE SERVER ID]",
+            "database.server.name": "[UNIQUE DATABASE SERVER NAME]",
+            "database.include.list": "[DATABASES TO CONNECT]",
+            "database.history.kafka.bootstrap.servers": "[KAFKA HOSTNAME:PORT]",
+            "topic.prefix": "[DATABASE SERVER NAME]",
+            "schema.history.internal.kafka.topic": "schema-changes.[DATABASE SERVER NAME].internal",
+            "schema.history.internal.kafka.bootstrap.servers": "[KAFKA HOSTNAME:PORT]",
+            "include.schema.changes": "true",
+            "decimal.handling.mode": "string",
+        }
+    }'
+```
+
+:::note
+Refer to the [secret management](../../enterprise/kubernetes-guides/secret-management) guide
+to externalize secrets such as DBMS passwords via Kubernetes.
+:::
+
+## Step 3: Create Feldera input connector
+
+Configure an input connector for each Feldera SQL table that must ingest changes from Debezium.
+Use the `kafka_input` transport with either `json` or `avro` format. Debezium automatically
+creates a Kafka topic for each database table.
+
+### JSON
+
+When using JSON encoding, make sure to set the following connector properties:
+
+* `"update_format": "debezium"`
+* `"json_flavor"` depending on the:
+  * For MySQL and MariaDB, set `"json_flavor": "debezium_mysql"`
+  * For all other databases set `"json_flavor": "debezium_postgres"`
+
+Configure a Feldera connector to ingest changes from a Postgres DB via a JSON-encoded Kafka topics:
+
+```sql
+CREATE TABLE my_table (
+   ... -- columns omitted
```

Comment:
```suggestion
   example_field INT
```

which has the nice effect that this compiles out of the box (I want to make a script that checks all our examples in docs against the sandbox)